src/v8/v8_context_scope.rs

```diff
@@ -252,6 +322,20 @@ impl<'isolate_scope, 'isolate> V8ContextScope<'isolate_scope, 'isolate> {
     }
 }
 
+impl<'isolate_scope, 'isolate> AsRef<V8Isolate> for V8ContextScope<'isolate_scope, 'isolate> {
+    fn as_ref(&self) -> &V8Isolate {
```

Comment:
I am not sure the the Ref of `V8ContextScope` should be the isolate

Reply:
It sounds more reasonable AsRef for `V8ContextScope` will return reference to `V8Context` and AsRef for `V8IsolateScope` will return an `Isolate` no?

Reply:
Got it, thank you for the explanation.